tests/EndToEndBuildTests/BasicProvider/TestBasicProvider.cmd

```diff
@@ -34,16 +34,16 @@ rem
 if not '%NUGET_PACKAGES%' == '' rd %NUGET_PACKAGES%\BasicProvider /s /q
 taskkill /F /IM dotnet.exe
 
-echo dotnet pack BasicProvider\BasicProvider.fsproj -o %~dp0artifacts -c %configuration% -v minimal -p:FSharpTestCompilerVersion=net40
-     dotnet pack BasicProvider\BasicProvider.fsproj -o %~dp0artifacts -c %configuration% -v minimal -p:FSharpTestCompilerVersion=net40
+echo dotnet pack BasicProvider\BasicProvider.fsproj -o %~dp0artifacts -c %configuration% -v minimal -p:FSharpTestCompilerVersion=net40 /bl:one.binlog
+     dotnet pack BasicProvider\BasicProvider.fsproj -o %~dp0artifacts -c %configuration% -v minimal -p:FSharpTestCompilerVersion=net40 /bl:two.binlog
```

Comment:
The binlog file names shouldn't change here; the first line is just echoing what command is about to be executed.  Similar for the rest of the file.

Reply:
Oops, that is when I was trying to figure out why it failed :-)